.jules/palette.md

```
@@ -0,0 +1,3 @@
+## 2024-04-25 - Context-Aware TUI Help Footers
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)